### PR TITLE
refactor(app): update connection result success case

### PR DESCRIPTION
**Pull request by koji**
# Overview
update connection result success case and test

# Test Plan
- Go to `Network setup menu` from ODD menu / Unboxing flow 
- Tap `Wi-Fi`
- Select a ssid / `Join other network` and try to connect to a network
- Tap `View network details` -> shows Network details modals
- Tap Continue -> go to RobotSoftware update 

# Changelog
- update WifiConnection Details and modify its test

# Review requests

# Risk assessment
low

### CODE REVIEW

File: app/src/App/OnDeviceDisplayApp.tsx [hunk 1]
```diff
@@ -244,7 +244,7 @@ export const OnDeviceDisplayApp = (): JSX.Element => {
   }, [dispatch, isIdle, usersBrightness])
 
   return (
-    <ApiHostProvider hostname="localhost">
+    <ApiHostProvider hostname="192.168.0.107">
```
Comment (jerader) [inline app/src/App/OnDeviceDisplayApp.tsx]:
```suggestion
    <ApiHostProvider hostname="localhost">
```

File: app/src/organisms/OnDeviceDisplay/SetupNetwork/WifiConnectionDetails.tsx [hunk 2]
```diff
@@ -1,235 +1,130 @@
...
   ALIGN_CENTER,
   JUSTIFY_CENTER,
-  PrimaryButton,
   BORDERS,
 } from '@opentrons/components'
 
 import { StyledText } from '../../../atoms/text'
-import { getNetworkInterfaces, fetchStatus } from '../../../redux/networking'
+import { MediumButton } from '../../../atoms/buttons'
+import { NetworkDetailsModal } from '../RobotSettingsDashboard/NetworkSettings/NetworkDetailsModal'
```
Comment (jerader) [inline app/src/organisms/OnDeviceDisplay/SetupNetwork/WifiConnectionDetails.tsx]: import can go lower

File: app/src/organisms/OnDeviceDisplay/SetupNetwork/__tests__/WifiConnectionDetails.test.tsx [hunk 3]
```diff
@@ -63,27 +67,32 @@ describe('WifiConnectionDetails', () => {
...
+    getByText('mock NetworkDetailsModal')
   })
 
   it('when clicking Check for updates button, should call mock function', () => {
-    const [{ getByRole }] = render(props)
-    const button = getByRole('button', { name: 'Check for updates' })
-    fireEvent.click(button)
+    const [{ getByText }] = render(props)
+    const button = getByText('Continue')
+    button.click()
```
Comment (jerader) [inline app/src/organisms/OnDeviceDisplay/SetupNetwork/__tests__/WifiConnectionDetails.test.tsx]: just a suggestion, you don't have to do this!

```suggestion
   getByText('Continue').click()
```

File: app/src/organisms/OnDeviceDisplay/SetupNetwork/__tests__/WifiConnectionDetails.test.tsx [hunk 4]
```diff
@@ -63,27 +67,32 @@ describe('WifiConnectionDetails', () => {
...
+    getByText('Wi-Fi')
+    getByText('Successfully connected to mockWifi!')
+    getByText('View network details')
+    getByText('Continue')
+  })
+
+  it('should render network details when tapping view network details', () => {
+    const [{ getByText }] = render(props)
+    const button = getByText('View network details')
+    button.click()
```
Comment (jerader) [inline app/src/organisms/OnDeviceDisplay/SetupNetwork/__tests__/WifiConnectionDetails.test.tsx]:
```suggestion
    getByText('View network details').click()
```

**Review by jerader — APPROVED** [review]
just some cleanup comments but otherwise lgtm

File: app/src/App/OnDeviceDisplayApp.tsx (hunk 1 above)
Comment (koji) [reply app/src/App/OnDeviceDisplayApp.tsx]: oh my bad.
thank you.

File: app/src/organisms/OnDeviceDisplay/SetupNetwork/__tests__/WifiConnectionDetails.test.tsx (hunk 3 above)
Comment (koji) [reply app/src/organisms/OnDeviceDisplay/SetupNetwork/__tests__/WifiConnectionDetails.test.tsx]: okay, I will switch to that way.

**Pull request merged**